packages/venia-ui/lib/components/CheckoutPage/ItemsReview/item.js

```diff
@@ -13,18 +14,24 @@ const Item = props => {
         product,
         quantity,
         configurable_options,
-        isHidden
+        isHidden,
+        configurableThumbnailSource
     } = props;
     const classes = mergeClasses(defaultClasses, propClasses);
     const className = isHidden ? classes.root_hidden : classes.root;
-
+    const configured_variant = configuredVariant(configurable_options, product);
     return (
         <div className={className}>
             <Image
                 alt={product.name}
                 classes={{ root: classes.thumbnail }}
                 width={100}
-                resource={product.thumbnail.url}
+                resource={
```

Comment:
Same here, I would suggest using the ternary operator.

Reply:
Done